tests/testproject/TestProject.IntegrationServiceA/Program.cs

```diff
@@ -29,6 +29,10 @@
 {
     builder.AddNpgsqlDataSource("postgresdb");
 }
+if (!resourcesToSkip.Contains(TestResourceNames.efnpgsql))
+{
+    builder.AddNpgsqlDbContext<NpgsqlDbContext>("postgresdb");
```

Comment:
One interesting thing here is that if I set "resourcesToSkip" to just have `TestResourceNames.postgres`, this test is going to fail because there won't be a "postgresdb" connection.

Maybe we should think about just one flag "disableSlowResources", which just turns off Oracle and Cosmos. (Can be a separate PR).